src/ui/input/input.stories.tsx

```diff
@@ -23,7 +23,7 @@ const Template: ComponentStory<typeof Input> = args => {
         <Input
           {...args}
           value={state}
-          onChange={e => setState(e.target.value)}
+          onChange={(e: any) => setState(e.target.value)}
```

Comment:
We can import `ChangeEvent` from react.
So type will be `(e: ChangeEvent<HTMLInputElement>)`

Reply:
fixed